kalite/topic_tools/tests/utils_tests.py

```diff
@@ -0,0 +1,47 @@
+from mock import patch, MagicMock
+
+import kalite.topic_tools as mod
+from kalite.testing.base import KALiteTestCase
+
+
+DUMMY_STRING = "maize"
+TRANS_STRING = "trans"
+
+
+ugettext_dummy = MagicMock(return_value=DUMMY_STRING)
+
+
+@patch.object(mod, '_', new=ugettext_dummy)
+class SmartTranslateItemDataTests(KALiteTestCase):
```

Comment:
Of course if you made the changes I suggested so that `'content'` is more thoroughly searched for then this test would fail.